VRDR/DeathRecord_submissionProperties.cs

```diff
@@ -2091,6 +2136,11 @@ public string DateOfBirth
                     Extension withinCityLimits = new Extension();
                     withinCityLimits.Url = ExtensionURL.WithinCityLimitsIndicator;
                     withinCityLimits.Value = DictToCoding(value);
+                    // Coding coding = DictToCoding(value);
```

Comment:
Does the DictToCoding function handle setting the null value correctly?

remove commented out code if not needed